contrib/network/connection_balance/dlb/source/connection_balancer_impl.cc

```diff
@@ -40,25 +40,21 @@ DlbConnectionBalanceFactory::createConnectionBalancerFromProto(
 #ifdef DLB_DISABLED
   throw EnvoyException("X86_64 architecture is required for Dlb.");
 #else
-  int device_id = 0;
+  int device_id = dlb_config.id();
```

Comment:
Could this be a negative number? Should the config be rejected if it is?

Reply:
Here the id is `uint32`, which cannot be negative, refer to https://stackoverflow.com/questions/765916/is-there-ever-a-good-time-to-use-int32-instead-of-sint32-in-google-protocol-buff